server/routerlicious/packages/lambdas/src/scribe/lambda.ts

```diff
@@ -35,8 +35,12 @@ import {
     IPartitionLambda,
     INackMessagesControlMessageContents,
 } from "@fluidframework/server-services-core";
+import { BaseTelemetryProperties, CommonProperties,
```

Comment:
nit: put `BaseTelemetryProperties`, `CommonProperties` and others on separate lines, like

```
import {
    BaseTelemetryProperties,
    CommonProperties,
    Lumber,
    LumberEventName,
    Lumberjack 
} from "@fluidframework/server-services-telemetry";
```